docs/formazione/README.md

```diff
@@ -0,0 +1,156 @@
+
+Facciamo un esempio: con JavaScript possiamo sviluppare _siti web_ ma non solo, grazie alla versatilità del linguaggio possiamo ad oggi spaziare su tantissimi altri argomenti dalle **Mobile App** ai **Videogiochi**, addirittura con JavaScript oggi possiamo fare qualcosa che fino a qualche anno fa sembrava impossibile: il _backend_.
+
+"Mi stai dicendo che se voglio fare il _Game Developer_ devo prima diventare un **programmatore web**?"
+Assolutamente no.
+
+Se si hanno le idee chiare sin da subito allora la scelta del percorso formativo è molto più semplice, se si hanno invece le idee confuse, inizialmente, allora partire dallo sviluppo web è la strada migliore in quanto più accessibile.
+
+"Mi stai dicendo che è più facile fare i siti web piuttosto che i giochi?"
+Assolutamente no.
+
+Il concetto è che fare siti web è più accessibile sia in termini di **risorse tecnologiche** che in termini di **know how** di partenza, per un neofita, rispetto a fare videogiochi.
+
+"Ok, ipotizziamo che voglia partire nell'imparare a fare siti web, come scelgo il percorso giusto?"
+Non c'è una risposta corretta o sbagliata a questa domanda, tutto dipende dal proprio approccio che si ha con lo studio, ma subentrano anche altri fattori non da poco come ad esempio il tempo (inteso sia come tempo a disposizione per studiare e sia come quantità di tempo necessaria per raggiungere lo scopo), oppure gli sbocchi lavorativi.
+
+## Prima fase: come approcciare lo studio
+
+Ci sono percorsi formativi che lasciano maggiore autonomia e percorsi formativi che invece seguono più assiduamente i propri alunni.
+Non c'è un giusto o sbagliato, l'importante è tenere a mente un altro concetto fondamentale: il programmatore non smette mai di imparare, e la maggior parte delle volte impara da solo.
+Personalmente mi sento di consigliare, almeno nella fase iniziale, di essere seguiti ed essere curiosi.
+Essere seguiti ci permette di avere una guida, un supporto (anche morale e non solo tecnico), qualcuno che indirizza e che spiega all'inizio è molto meglio piuttosto che chiedersi se stiamo approcciando correttamente o meno ad un problema o ad una soluzione.
+Essere curiosi invece ci permette di imparare ad essere autonomi e soprattutto, cosa molto improtante, a trovare la nostra strada.
+
+**Google è il nostro migliore amico**.
+
+## Seconda fase: il tempo
+
+Alcuni percorsi di formazione propongono 3 mesi altri 6 mesi, altri invece hanno dei pacchetti personalizzati.
+Alcuni sono costituiti da classi e le lezioni si tengono in presenza, altri invece completamente da remoto con lezioni live oppure on-demand.
+Anche qui non c'è una formula magica.
+
+Diventare programmatore non è semplice, essere programmatore è una questione di **mindset**, quindi serve del tempo per cambiare il nostro modo di approcciare e di vedere le cose.
```

Comment:
Ricordiamoci sempre la gender neutrality, io in primis me la perdo quindi mi permetto di indicarla se la noto ahah

Reply:
Corretto, vero @serenasensini ?